digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/ProjectService.scala

```diff
@@ -217,17 +222,15 @@ class ProjectService(roadAddressService: RoadAddressService, roadLinkService: Ro
     }).asInstanceOf[Seq[ProjectLink]]
   }
 
-  def changeDirection(projectLink:Seq[Long]): String = {
+  def changeDirection(projectId : Long, roadNumber : Long, roadPartNumber : Long): String = {
```

Comment:
It'd be better to return Option[String] as an error message: None for no errors, otherwise Some("errormessage")